.env

```diff
@@ -0,0 +1 @@
+DEVISE_JWT_SECRET_KEY=9294bd736596a6c435adb912185e8ddd4c7f91cd199d519e462f06d3255a20a99779122224cd796c392a3b460e8b12e18875250bf05283f83fbf04c3b9909822
```

Comment:
Secret Key should not be published on GitHub because it can cause security issues in the future and malicious users can get unauthorized access. Please add `.env` in `.gitignore` file so that It won't be published.